base/doc/ltnews39.tex

```diff
@@ -151,6 +151,16 @@ \section{New or improved commands}
 
 \section{Code improvements}
 
+\subsection{Loading packages at the top level}
+Packages should only be loaded with \cs{usepackage} or \cs{RequirePackage}
```

Comment:
Mind referring to `\RequirePackageWithOptions` as well?

Reply:
could be I guess

Reply:
added